draft-ietf-privacypass-batched-tokens.md

```diff
@@ -409,19 +410,19 @@ struct {
 If the FinalizeBatch function fails, the Client aborts the protocol. Token
 verification works exactly as specified in {{RFC9578}}.
 
-# Arbitrary Batched Token Issuance
+# Arbitrary Batched Token Issuance {#batched-arbitrary}
```

Comment:
note: this is also introduced in #34

Reply:
I guess it's only fair to re-align #34 in that case